From 8efdd6a692d2dad13f990695c3eaec6b9c477e4e Mon Sep 17 00:00:00 2001
From: Loren Huang <b02279@freescale.com>
Date: Fri, 14 Sep 2012 16:08:20 +0800
Subject: [PATCH 2041/2463] ENGR00224338 mx6:gpu:Make gpu as buildin driver by
 default

Make gpu as build-in driver to simplify gpu test and usage.

Signed-off-by: Loren Huang <b02279@freescale.com>
---
 arch/arm/configs/imx6_defconfig  |    2 +-
 arch/arm/configs/imx6s_defconfig |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index f9a4eac..173fcc8 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -2305,7 +2305,7 @@ CONFIG_MXC_MLB150=m
 #
 # MXC Vivante GPU support
 #
-CONFIG_MXC_GPU_VIV=m
+CONFIG_MXC_GPU_VIV=y
 
 #
 # ANATOP_THERMAL
diff --git a/arch/arm/configs/imx6s_defconfig b/arch/arm/configs/imx6s_defconfig
index 5f66b3e..95e07d5 100644
--- a/arch/arm/configs/imx6s_defconfig
+++ b/arch/arm/configs/imx6s_defconfig
@@ -2262,7 +2262,7 @@ CONFIG_MXC_MLB150=m
 #
 # MXC Vivante GPU support
 #
-CONFIG_MXC_GPU_VIV=m
+CONFIG_MXC_GPU_VIV=y
 
 #
 # ANATOP_THERMAL
-- 
1.7.7.4

